doc/tutorials/enable_s5.rst

```diff
@@ -31,131 +31,197 @@ The diagram below shows the overall architecture:
+         $ scp misc/services/life_mngr/user_vm_shutdown.py root@<target board address>:~/
 
-#. For LaaG and RT-Linux VMs, run the lifecycle manager daemon:
+      .. note:: user_vm_shutdown.py is only needed to be copied into Service VM.
 
-   a. Use these commands to build the lifecycle manager daemon, ``life_mngr``.
+   #. Edit options in ``/etc/life_mngr/life_mngr.conf`` in the Service VM.
 
       .. code-block:: none
 
-         $ cd acrn-hypervisor
-         $ make life_mngr
+         VM_TYPE=service_vm
+         VM_NAME=Service_VM
+         DEV_NAME=tty:/dev/ttyS8,/dev/ttyS9,/dev/ttyS10,/dev/ttyS11,/dev/ttyS12,/dev/ttyS13,/dev/ttyS14
+         ALLOW_TRIGGER_S5=/dev/ttySn
+
+      .. note:: The mapping between User VM ID and communication serail device name (/dev/ttySn)
+                  in the /etc/serial.conf. If /dev/ttySn is configured in the  ``ALLOW_TRIGGER_S5``,
+                  this means system shutdown is allowed to be triggered in the the corresponding User VM.
 
-   #. Copy ``life_mngr`` and ``life_mngr.service`` into the User VM:
+   #. Edit options in ``/etc/life_mngr/life_mngr.conf`` in the User VM.
 
       .. code-block:: none
 
-         $ scp build/misc/services/life_mngr root@<test board address>:/usr/bin/life_mngr
-         $ scp build/misc/services/life_mngr.service root@<test board address>:/lib/systemd/system/life_mngr.service
+         VM_TYPE=user_vm
+         VM_NAME=<User VM name>
+         DEV_NAME=tty:/dev/ttyS1
+         #ALLOW_TRIGGER_S5=/dev/ttySn
 
-   #. Use the below commands to enable ``life_mngr.service`` and restart the User VM.
+      .. note:: The User VM name in this configuration file should be consistent with the VM name in the
+               launch script for the Post-launched User VM or the VM name which is specified in the hypervisor
+               scenario XML for the Pre-launched User VM.
+
+   #. Use the following commands to enable ``life_mngr.service`` and restart the Service VM and User VMs.
 
       .. code-block:: none
 
          # chmod +x /usr/bin/life_mngr
          # systemctl enable life_mngr.service
          # reboot
 
+      .. note:: For the Pre-launched User VM, need restart Lifecycle Manager service manually
+         after Lifecycle Manager in service VM starts.
+
 #. For the WaaG VM, run the lifecycle manager daemon:
 
-   a) Build the ``life_mngr_win.exe`` application::
+   a) Build the ``life_mngr_win.exe`` application and ``s5_trigger_win.py``::
 
         $ cd acrn-hypervisor
         $ make life_mngr
 
-      .. note:: If there is no ``x86_64-w64-mingw32-gcc`` compiler, you can run ``sudo apt install gcc-mingw-w64-x86-64``
-         on Ubuntu to install it.
+      .. note:: If there is no ``x86_64-w64-mingw32-gcc`` compiler, you can run
+               ``sudo apt install gcc-mingw-w64-x86-64`` on Ubuntu to install it.
+
+   #) Copy ``s5_trigger_win.py`` into the WaaG VM.
+
+      .. note:: If there is no ``python 3.8.10``, you can install it on the WaaG VM.
```

Comment:
We should rephrase this to indicate the user must install Python in his WaaG.

Reply:
Thanks, will update this.